__tests__/helpers/openingHoursHelper.test.js

```diff
@@ -0,0 +1,215 @@
+import { getReadableDay, isOpen } from '../../src/helpers';
+
+const todayString = getReadableDay(new Date());
+
+const FAKE_NOW_DATE = new Date();
+FAKE_NOW_DATE.setHours(12);
+FAKE_NOW_DATE.setMinutes(0);
+FAKE_NOW_DATE.setSeconds(0);
+FAKE_NOW_DATE.setMilliseconds(0);
+
+const isOpenWithFakeTimeDateTime = (openingHours) => isOpen(openingHours, FAKE_NOW_DATE);
+
+describe('testing the opening times handling', () => {
+  test('single opening intervals work as expected', () => {
```

Comment:
how come that you are using `describe/test` instead of `describe/it` like we have in the rest of our tests?